tutorial/practicals/LFRic/building_code/4_psydata/Makefile.inc

```diff
@@ -65,7 +65,9 @@ PSYCLONE_RELPATH = ../../../../..
 LFRIC_PATH = $(PSYCLONE_RELPATH)/src/psyclone/tests/test_files/dynamo0p3/infrastructure
 LFRIC_NAME = lfric_netcdf
 LFRIC_LIB = $(LFRIC_PATH)/lib$(LFRIC_NAME).a
-F90FLAGS += -I$(LFRIC_PATH) $$(nf-config --fflags)
+
+# This will add the required include flags to F90FLAGS
+include $(LFRIC_PATH)/lfric_include_flags.inc
```

Comment:
I did not have to modify the Makefile in solutions, since they all include `Makefile.inc`, which I had updated.

And yes, jenkins does compile the tutorials, and this branch all worked. It took me a while to find out what was likely happening: I think for whatever reason the compiler created a .mod and .o file with inconsistent times - the .mod file was newer than the .o file. This then triggered the implicit modula rule in makefiles: if there is a modula .mod file, it can be used to create the .o file. I could reproduce the problem by explicitly touching the .mod file. I've added an explicit rule that .o depends on the .mod, which then reliably avoided the implicit rule for me.

Could you see if this fixes the problem for you?

Reply:
Yep, that seems to be fine now - thanks :-)